cpp/velox/compute/WholeStageResultIterator.cc

```diff
@@ -307,6 +308,18 @@ void WholeStageResultIterator::collectMetrics() {
       metrics_->get(Metrics::kIoWaitTime)[metricIndex] = runtimeMetric("sum", second->customStats, kIoWaitTime);
       metrics_->get(Metrics::kPreloadSplits)[metricIndex] =
           runtimeMetric("sum", entry.second->customStats, kPreloadSplits);
+
+      // Update the Write metrics.
+      for (int i = 0; i < task_->taskStats().pipelineStats.size(); ++i) {
+        auto operatorStats = task_->taskStats().pipelineStats.at(i).operatorStats;
+        for (int j = 0; j < operatorStats.size(); ++j) {
+          auto operatorStat = operatorStats.at(j);
+          if (operatorStat.operatorType == "TableWrite") {
```

Comment:
If there are several `TableWrite` operators, how to ensure the get metrics are matched?